tests/query/processors/test_org_rate_limiter.py

```diff
@@ -102,3 +102,33 @@ def test_org_rate_limit_processor_overridden(
     assert rate_limiter.bucket == str(org_id)
     assert rate_limiter.per_second_limit == 5
     assert rate_limiter.concurrent_limit == 10
+
+
+def test_namespaced_rate_limit() -> None:
+    query = Query(
+        QueryEntity(EntityKey.EVENTS, EntityColumnSet([])),
+        selected_columns=[SelectedExpression("column2", Column(None, None, "column2"))],
+        condition=binary_condition(
+            ConditionFunctions.EQ,
+            Column("_snuba_org_id", None, "org_id"),
+            Literal(None, 1),
+        ),
+    )
+    org_id = 1
+    settings = HTTPQuerySettings()
+    ps_key = f"org_per_second_limit_{org_id}"
+    ct_key = f"org_concurrent_limit_{org_id}"
+    state.set_config(ps_key, 5)
+    state.set_config(ct_key, 10)
+
+    OrganizationRateLimiterProcessor("org_id").process_query(query, settings)
```

Comment:
I think this comment would help make clear what the test is testing. It took me a few re-reads to understand it otherwise

```suggestion
    # By running the rate limiter processor, the keys should be ported over to the 
    # rate_limit_config_hash namespace
    OrganizationRateLimiterProcessor("org_id").process_query(query, settings)
```